src/index.js

```diff
@@ -1 +1,9 @@
+import artistsAPI from './modules/getArtistAPI';
+import createCard from './modules/homeUI';
 import './style.css';
+
+document.addEventListener('DOMContentLoaded', async () => {
+  const artists = await artistsAPI.getArtist();
+  console.log(artists);
```

Comment:
Please remove the console.log line in your code. This line is only used for debugging &#9935;

![](https://giphy.com/clips/therokuchannel-roku-the-channel-50-states-of-fright-uCp2H6Il7ssfGUDFVk)

Reply:
@wayungi Thank your for spotting this out :pray: